tests/main/interfaces-physical-memory-observe/task.yaml

```diff
@@ -0,0 +1,52 @@
+summary: Ensure that the physical memory observe interface works.
+
+details: |
+    The physical-memory-observe interface allows to read the physical memory.
+
+    The test-snapd-physical-memory-observe snap checks that /dev/mem can be read
+    and the interface can be connected and disconnected. 
+
+# As the ubuntu kernels are configured with CONFIG_STRICT_DEVMEM=y, this interface is
+# validated just in the other supported systems.
+systems: [-ubuntu-*]
+
+prepare: |
+    echo "Given the physical-memory-observe snap is installed"
+    snap try $TESTSLIB/snaps/test-snapd-physical-memory-observe
```

Comment:
Should we snap remove it in restore?

Reply:
it is being done automatically in the prepare. The reset script is called and it is deleting all the installed snaps and cleaning up the env.